straxen/plugins/merged_s2s/merged_s2s.py

```diff
@@ -16,7 +16,7 @@ class MergedS2s(strax.OverlapWindowPlugin):
     """Merge together peaklets if peak finding favours that they would form a single peak
     instead."""
 
-    __version__ = "1.0.2"
+    __version__ = "1.1.0"
```

Comment:
Presumably changes in this file are all from PR #1299?

Reply:
Yes, they are. Sorry that it is a bit messy, but I thought you guys would look first at #1299